### DISCUSSION TITLE
Is it possible to make sp-slider vertical ?

### DISCUSSION THREAD

**Discussion by flatmax** [adobe/spectrum-web-components#3122] [Q&A] [answered]
I would like a vertical slider. How do I style sp-slider to make it vertical ?

**Comment by Westbrook** [comment, marked as answer]
We don't currently have support for this, see: https://github.com/adobe/spectrum-web-components/issues/2391 It's something we'd love to add, if you were interested in submitting a PR.

**Reply by flatmax** [reply]
got it - thanks
